src/flags.ts

```diff
@@ -0,0 +1,113 @@
+      "It looks like the proposed changes may require planning permission.",
+    ],
+    PRIOR_APPROVAL: [
+      "Prior approval",
+      "#FCFF58",
+      "It looks like the proposed changes do not require planning permission, however the applicant must apply for Prior Approval before proceeding.",
+    ],
+    "PP-NOTICE": [
+      "Notice",
+      "#CAFB8B",
+      "It looks like the proposed changes may not require planning permission, however the applicant must provide notice to the planning authority before proceeding.",
+    ],
+    NO_APP_REQUIRED: [
+      "Permitted development",
+      "#82E7A1",
+      "It looks like the proposed changes may fall within the rules for Permitted Development and therefore would not need planning permission.",
+    ],
+    "PP-NOT_DEVELOPMENT": [
+      "Not development",
+      "#FFF",
+      "It looks like the proposed changes may not fall within the legal definition of ‘development’, and therefore would not require planning permission.",
+    ],
+  },
+  "Listed building consent": {
+    "LB-MISSING_INFO": ["Missing information", "#EAEAEA"],
+    "LB-REQUIRED": ["Required", "#76B4E5"],
+    "LB-DE_MINIMIS": ["De minimis", "#CEE4F6"],
+    "LB-NOT_REQUIRED": ["Not required", "#FFF"],
+  },
+  "Works to trees & hedges": {
+    "TR-MISSING_INFO": ["Missing information", "#EAEAEA"],
+    "TR-REQUIRED": ["Required", "#9ED9B0"],
+    "TR-DE_MINIMIS": ["De minimis", "#E5F5EA"],
+    "TR-NOT_REQUIRED": ["Not required", "#FFF"],
+  },
+  "Demolition in a conservation area": {
+    "DC-MISSING_INFO": ["Missing information", "#EAEAEA"],
+    "DC-REQUIRED": ["Required", "#9ED9B0"],
+    "DC-DE_MINIMIS": ["De minimis", "#E5F5EA"],
+    "DC-NOT_REQUIRED": ["Not required", "#FFF"],
+  },
+  "Planning policy": {
+    PO_MISSING_INFO: ["Missing information", "#EAEAEA"],
+    LIKELY_FAIL: ["Fails to meet policy", "#FF7F78"],
+    EDGE_CASE: ["Edge case", "#FFA05B"],
+    LIKELY_PASS: ["Meets policy", "#AAEB61"],
+  },
+  "Community infrastructure levy": {
+    CO_MISSING_INFO: ["Missing information", "#EAEAEA"],
+    CO_EXEMPTION_VOID: ["Exemption void", "#CDB1C2"],
+    CO_EXEMPT: ["Exempt", "#FFDEF2"],
+    CO_RELIEF_VOID: ["Relief void", "#A4698C"],
+    CO_RELIEF: ["Relief", "#FFA4DA"],
+    CO_LIABLE: ["Liable", "#FF0099"],
+    CO_NOT_LIABLE: ["Not liable", "#FFF"],
+  },
+};
+
+export const flatFlags: Array<Flag> = [];
+
+export type FlagSet = keyof typeof categoriesAndFlags;
+
+const parsedFlags: Record<FlagSet, { [id: string]: Flag }> = Object.entries(
+  categoriesAndFlags
+).reduce((acc: Record<string, any>, [category, flags]) => {
+  acc[category] = Object.entries(flags).reduce(
+    (acc: Record<string, Flag>, [id, [text, bgColor, officerDescription]]) => {
+      // loop through all the flags and add a text color which is
+      // white if it's a dark background or black if it's light
+      const color = "#000";
+        // mostReadable(bgColor, ["#000", "#FFF"])?.toHexString() || "#000";
```

Comment:
I guess `color` isn't needed for BOPS, is that right? Can we remove this commented out code and comments?

Reply:
Yep, I had originally kept this file the same as `planx-new` and just commented out this line to avoid adding an unnecessary package dependency here - but realise how that might be confusing. Fully removed references to `color` and `bgColor` now as those aren't required in export data formats.